ynr/apps/parties/migrations/0014_add_last_updated.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations
+import django.utils.timezone
+import django_extensions.db.fields
+
+
```

Comment:
@michaeljcollinsuk I think I need to add a dependency there, but I'm unsure which.

Reply:
Looks like something has gone amiss here, we dont need to add a `last_updated` field to the `Party` model. And I can't see any change to the `Party` model that would have caused this migration to be generated either. So you can delete this file (hopefully you didn't run `manage.py migrate` to apply it yet?) as the `modified` field will already be on the `Party` model via the `TimeStampedModel`, which is what will be used to populate the `last_updated` value on the serialised. So it should just be a case of adding the `last_updated` field to the `PartySerializer` - in the same way you have done for other serializers, where it is defined like:
```
last_updated = serializers.DateTimeField(source="modified")
```

Reply:
Ah ok, I didn't realise it would already be made available. I didn't run any migrations so will just delete the file. I was confused by an error, but it was actually an import issue and I have since solved it.

Reply:
So now its available on the serializer, just missing the ability to filter on the field. So that is same practice as the others - add a `PartyFilter` that can inherit from the mixing you added, then adding it to the `PartyViewSet`